dist/src/main/java/io/camunda/commons/clustering/AtomixClusterConfiguration.java

```diff
@@ -27,6 +27,8 @@ public AtomixClusterConfiguration(final ClusterConfig config) {
 
   @Bean(destroyMethod = "stop")
   public AtomixCluster atomixCluster() {
-    return new AtomixCluster(config, Version.from(VersionUtil.getVersion()));
+    final var atomixCluster = new AtomixCluster(config, Version.from(VersionUtil.getVersion()));
+    atomixCluster.start();
```

Comment:
❓ Where was this started before? For example in the broker bootstrap right? 🤔 How does this affect now the bootstrapping, is this an issue?

Reply:
* When the app gets started with the profile `broker`, the atomix cluster is started as one of the startup steps.
* When the app is started with the profile `gateway`, then the atomix cluster is started in the `GatewayModuleConfiguration` in the `dist` module.

Now, with that change, the atomix cluster gets started before the broker or the gateway actually starts it. Because, when the app is started with the profile `operate` then the broker and the gateway itself are not started, but the atomix cluster still must be started so that the app can join the cluster and forward v2 requests to the broker where applicable.

> How does this affect now the bootstrapping, is this an issue?
I went through the different paths; to me, they do not affect the bootstrapping. It starts the Atomix cluster, and if the broker once again tries to start it (when applying the startup steps), it's a noop because `start()` ensures that if it is already started, it won't be started again. Also, I couldn't spot any place where it must be started only after something different happened. Yes, the broker startup steps provide a specific order, but it doesn't there is nothing that relies on it. The same applies to the gateway startup procedure.

I checked if I could refactor that part so that it starts only in one place, but it seemed to require a bigger refactoring. So I went this path, and as I said, debugged the code paths and tested them locally but couldn't spot any issues.

Reply:
Thanks for checking. 👍🏼 I remember some years ago we had issues with twice starting services (which would break, etc.) also depending on the order. But If you see no actual issue, I'm fine. but yes would be good to refactor this at somepoint.